src/layouts/Layout.jsx

```diff
@@ -33,9 +33,12 @@ const Layout = ({ ariaLabel, children }) => {
         minHeight: '100vh'
       }}
     >
+      <NavBarSkipLink />
       <NavBar />
       {session.info.isLoggedIn && <Breadcrumbs />}
-      <main>{children}</main>
+      <main id="main-content" tabIndex={-1}>
```

Comment:
Also, it doesn't need a tabIndex of -1. That's for if we want to allow _focusing_, but we are just navigating to that area of the page.

Reply:
😃  I had done this then reverted it...
Added `<Main />` with no `tabIndex`.

Reply:
> 😃 I had done this then reverted it... Added `<Main />` with no `tabIndex`.

Yeah my question is why the suggestion and implementation to make it it's own component, instead of just using native html `<main id='main-content'>`

Reply:
I'm just thinking that wrapping a `main` that essentially requires a specific `id` to work properly that we're going to have in various places around our app into an abstracted component would be good. Then future devs wouldn't have to worry about putting in the `id` and there would be context around _why_ it is done this way, which is good for future education and avoiding regressions.